wercker.yml

```diff
@@ -1,34 +1,18 @@
-box: blisteringherb/scholarship@0.0.8
+box: blisteringherb/ds-wercker-box:mysql
```

Comment:
This doesn't block merge, but it seems like the "Docker way" of handling service dependencies is attaching a [separate container](http://devcenter.wercker.com/docs/services) rather than having it all bundled in one. Maybe something to look into in a future PR?